package.json

```diff
@@ -1,26 +1,51 @@
 {
     "name": "pact-gen-ts",
     "version": "0.0.1",
+    "description": "Generating pact files from typescript definitions",
+    "keywords": [
+        "pacts",
+        "contracts",
+        "tests",
+        "typescript"
+    ],
     "main": "./lib/index.js",
+    "bin": {
+        "pact-gen-ts": "./bin/pact-gen-ts.js"
+    },
+    "files": [
+        "lib",
+        "bin"
+    ],
     "license": "MIT",
     "scripts": {
-        "build": "tsc -p tsconfig.json"
+        "build": "tsc -p tsconfig.json",
+        "release": "semantic-release"
+    },
+    "dependencies": {
+        "qs": "^6.10.1",
+        "ts-morph": "^10.0.2"
     },
     "devDependencies": {
+        "@semantic-release/changelog": "^5.0.1",
+        "@semantic-release/git": "^9.0.0",
```

Comment:
@semantic-release/changelog and @semantic-release/git can be removed, they are not used anywhere.

Reply:
ok, removed